packaging/setup_osx.py.in

```diff
@@ -0,0 +1,73 @@
+import sys
+import os.path
+
+from cx_Freeze import setup, Executable, hooks
+
+def load_scipy(finder, module):
+    finder.IncludePackage("scipy._lib")
+    finder.IncludePackage("scipy.misc")
+    finder.IncludePackage("scipy.sparse.csgraph._validation")
+
+hooks.load_scipy = load_scipy
+
+def load_pyqt5_qtquick(finder, module):
+    finder.IncludeModule("PyQt5.QtCore")
+    finder.IncludeModule("PyQt5.QtGui")
+    finder.IncludeModule("PyQt5.QtQml")
+    finder.IncludeModule("PyQt5.QtNetwork")
+    finder.IncludeModule("PyQt5._QOpenGLFunctions_2_0")
+    
+hooks.load_PyQt5_QtQuick = load_pyqt5_qtquick
+
+search_path = sys.path.copy()
+search_path.insert(1, "@EXTERNALPROJECT_INSTALL_PREFIX@/lib/python3.5/site-packages/")
+search_path.insert(2, "@CMAKE_INSTALL_PREFIX@/lib/python3.5/site-packages/")
+
+# Dependencies are automatically detected, but it might need
+# fine tuning.
+build_options = {
+#    "build_exe": "package",
+    "path": search_path,
+    "packages": [
+        "xml.etree",
+        "uuid",
+        "UM",
+        "cura"
+    ],
+    "include_files": [
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/bin/CuraEngine", ""),
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/lib/cura", "plugins"),
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/lib/uranium", "plugins"),
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/lib/python3.5/site-packages/UM/Qt/qml/UM", "qml/UM"),
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/share/cura/resources", "resources"),
+        ("@EXTERNALPROJECT_INSTALL_PREFIX@/share/uranium/resources", "resources"),
+        ("@CMAKE_INSTALL_PREFIX@/lib/qml/Qt", "qml/Qt"),
+        ("@CMAKE_INSTALL_PREFIX@/lib/qml/QtQml", "qml/QtQml"),
+        ("@CMAKE_INSTALL_PREFIX@/lib/qml/QtQuick", "qml/QtQuick"),
+        ("@CMAKE_INSTALL_PREFIX@/lib/qml/QtQuick.2", "qml/QtQuick.2"),
+        ("@CMAKE_BINARY_DIR@/qt.conf", ""),
+    ],
+    "excludes": [ ]
+}
+
+bdist_mac_options = {
+    "iconfile": "@CMAKE_SOURCE_DIR@/packaging/cura.icns",
+    "custom_info_plist": "@CMAKE_BINARY_DIR@/Info.plist",
+    "bundle_name": "Cura"
+}
+
+executables = [
+    Executable("@EXTERNALPROJECT_INSTALL_PREFIX@/bin/cura_app.py", targetName = "cura")
+]
+
+setup(
+    name="Cura",
+    version = "@CURA_VERSION_MAJOR@.@CURA_VERSION_MINOR@.@CURA_VERSION_PATCH@",
+    description = "3D Printing Software",
+    author = "Ultimaker B.V.",
+    url="http://software.ultimaker.com/",
+    license="GNU AFFERO GENERAL PUBLIC LICENSE (AGPL)",
```

Comment:
Code style: Whitespace around binary operators. Fixed this for you in 71602f2f9fc65177b0ff5d17b34ea41af8ecdab9.